speech/api/README.md

```diff
@@ -38,6 +38,21 @@ See the
 [Cloud Platform Auth Guide](https://cloud.google.com/docs/authentication#developer_workflow)
 for more information.
 
+### Install pip
```

Comment:
I would prefer you to consolidate these steps like the [app engine flex readme](https://github.com/GoogleCloudPlatform/python-docs-samples/tree/master/appengine/flexible#run-locally). Feel free to link to virtualenv and pip, but there's no need to give explicit instructions on installation.